dashboard/admin.py

```diff
@@ -115,8 +116,14 @@ def save_model(self, request, obj, form, change):
         return super(CourseAdmin, self).save_model(request, obj, form, change)
 
 class LogResource(resources.ModelResource):
+
+    # This field is used to get the username from the user model
+    username = Field(attribute='user__username', column_name='username')
```

Comment:
Optional: is the `column_name` here giving a custom name to the data field as part of the `auth_user.username`. If yes, will it make sense to name it as `user_uniq_id`?

Reply:
Yes that's the header in the CSV, otherwise it was `user__username`. Can change it to something else. This just matched up with the column in the other table.

Reply:
I'm not sure, it does represent the username column in the auth_user table. We could call it auth_user__username maybe or left it as user__username. I think for purposes of the CSV as a column header it's fine as the shorter username or the other way. I don't think any admin using this is going to care either way.